src/pages/_app.tsx

```diff
@@ -13,6 +15,15 @@ import { RootLayout } from "@/layouts"
 import { inter, mono } from "@/lib/fonts"
 
 const App = ({ Component, pageProps }: AppPropsWithLayout) => {
+  useEffect(() => {
+    if (!process.env.IS_PREVIEW_DEPLOY) {
+      init({
```

Comment:
Should we add `disableCookies: true` here too

https://github.com/SocialGouv/matomo-next?tab=readme-ov-file#disable-cookies-

Reply:
We were not doing that on prod. Open to do it though but we might prefer to have that discussion separately?